{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```
@@ -1,6 +1,10 @@
 # syntax=docker/dockerfile:1
 ARG PYTHON_VERSION={{ cookiecutter.python_version }}
+{%- if cookiecutter.processing_unit == "GPU"%}
+FROM radixai/python-gpu:cuda11.8-python$PYTHON_VERSION AS base
+{%- else %}
 FROM python:$PYTHON_VERSION-slim AS base
+{%- endif %}
```

In line with the previous comment, this would become:
```suggestion
FROM {{ cookiecutter.docker_image }} AS base
```